lost/src/main/java/com/mapzen/android/lost/api/GeofencingApi.java

```diff
@@ -6,6 +6,8 @@
 
 public interface GeofencingApi {
 
+  void addGeofences(GeofencingRequest geofencingRequest, PendingIntent pendingIntent);
```

Comment:
We should add this annotation here: `@RequiresPermission(anyOf = {ACCESS_COARSE_LOCATION, ACCESS_FINE_LOCATION})`